Syndication/AzureStack.MarketplaceSyndication.psm1

```diff
@@ -90,165 +83,1086 @@ function Sync-AzSOfflineMarketplaceItem {
                     })
             }
 
+            'solution' {
+                Write-output ([pscustomobject]@{
+                        Id          = $product.name.Split('/')[-1]
+                        Type        = "Solution"
```

Comment:
This is the only field that is different? Should not need to repeat all this code, just return one object and use a lookup table for the translation of "productKind" to "type display name"

Reply:
Oh I see that type "virtualMachine" also has a different value for the Version, is that correct / intentional?